### ISSUE TITLE
Match programs based only on their window title

### ISSUE THREAD

**Issue by guihkx** [zee-mzha/vibrantLinux#4]
Vibrant Linux currently uses a path-based mechanism to check if a program is running or not. And if I understand correctly, that is done by checking if the canonical path of `/proc/<pid>/exe` is on the list of the added programs.

Unfortunately, that doesn't work for sandboxed Flatpak apps/games. For example, here's what I get when I try to read the canonical path of CS:GO, which is currently running on Steam for Flatpak:

```
$ file /proc/$(pidof csgo_linux64)/exe
/proc/37005/exe: symbolic link to /home/gui/.local/share/Steam/steamapps/common/Counter-Strike Global Offensive/csgo_linux64
```

However, that path doesn't really exist outside of the sandbox:

```
$ realpath /proc/$(pidof csgo_linux64)/exe
realpath: /proc/37005/exe: No such file or directory
```

So, my idea is to add a much simpler, title-based, window matching feature. Support for partial or exact matches would be a nice addition as well.

Thanks!

**Comment by zee-mzha** [comment]
Thanks for the idea, this will be included in the v2.0.0 switch. This will probably involve a breaking change since i'd have to change how vibrantLinux saves its program data but the save format was in need of a remodel anyway so it all works out.

**Comment by zee-mzha** [comment]
done <3
